eslint.config.js

```diff
@@ -0,0 +1,34 @@
+import globals from 'globals';
```

Comment:
Is this required to have JS configuration, not static `.eslintrc`? It's much less readable, the globals can be hardcoded. I don't see much value in that.